app/grants/models/subscription.py

```diff
@@ -186,6 +186,14 @@ class Subscription(SuperModel):
     def negative(self):
         return self.is_postive_vote == False
 
+    @property
+    def match_amount(self):
+        return 4.4 # TODO: actually calc it
```

Comment:
s there a code sample that shows how to get the match estimate for a given contribution?
eg calculates it based upon the grants curve?
or should i wrote some code that does linear interpolation using hte match curve?

Reply:
you could do a linear curve

the estimation we have in DB is 
at 0 DAI => 12 DAI 
1 DAI => 40 DAI
10 DAI => 80 DAI

So if the user sends 5 DAI, we'd simply do a linear curve and write the prediction amount

Reply:
@nutrina this might be worth picking up if you've got time

Reply:
I've pushed these changes 👍